website-hugo/src/themes/covid19/assets/js/covid.js

```diff
@@ -4,6 +4,55 @@ var startDate = new Date();
 var endDate = new Date();
 var formState = {};
 
+var mymap = L.map('mapid').setView([38.7436214, -9.1953085], 13);
```

Comment:
would suggest around 47,11 and level 5 to cover europe

Reply:
I was even thinking on having different coordinates depending on the country. But yes, for now makes sense